pages/terra/contributing.mdx

```diff
@@ -66,10 +87,17 @@ Remember, it takes effort to create a package. If you ever need help, hop into
    - It is a custom file format for RPM packages.
    - This [RPM Packaging Guide] might help newbies with no prior experiences with RPM specs.
    - This [Spec file format] docs goes into the details of the spec file format.
+   - You could use `rust2rpm` for **Rust** packages.
```

Comment:
"could" flows weird, try can or may

you may wanna try

You can use *2rpm tools like:
- rust2rpm
- go2rpm

instead